src/YNCursesUI.cc

```diff
@@ -451,3 +451,26 @@ YWidget * YNCursesUI::askSendWidgetID()
 
     return 0;
 }
+
+void YNCursesUI::sendEvent( NCursesEvent event )
+{
+    // do not send anything if the events are globally blocked
+    // i.e. allow masking the events caused by changes from the code
+    if (eventsBlocked())
+    {
+        yuiDebug() << "Events blocked, ignoring event " << event << std::endl;
+        return;
+    }
+
+    NCDialog *dialog = dynamic_cast<NCDialog *>(NCDialog::currentDialog(false)); // don't throw
+
+    if (dialog)
+    {
+        yuiDebug() << "Sending event: " << event << std::endl;
```

Comment:
Are you sure you want to keep this logging active after the development phase? This might be very verbose.

How about
```C++
#define VERBOSE_EVENTS 0
...
...
#if VERBOSE_EVENTS
    yuiDebug() << ...
#endif